File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2714.py
n = int(input())

for i in range(n):
	l = input()

	if(l[0] == 'R' and l[1] == 'A' and len(l) == 20):
		des = False
		res = ""

		for j in range(2, len(l)):
			if l[j] != '0':
				des = True

			if(des):
				res += l[j]

		print(res)
		
	else: 
		print("INVALID DATA")

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2714.c
#include <stdio.h>
#include <string.h>

int main(){
	int n;
	scanf("%d", &n);

	while(n--){
		char l[64];
		scanf("%s", l);

		if(l[0] == 'R' && l[1] == 'A' && strlen(l) == 20){
			char res[64];
			int des = 0, j;

			for(j = 2; j < strlen(l); j++){
				if(l[j] != '0'){
					des = 1;
				}

				if(des == 1){
					printf("%c", l[j]);
				}

			}

			printf("\n");

		} else{
			printf("INVALID DATA\n");
		}

	}

	return 0;
}